module-cellular/at/UrcCusd.hpp

```diff
@@ -30,26 +32,17 @@ namespace at::urc
             OperationNotSupported,
             NetworkTimeOut
         };
-        Cusd(const std::string &urcBody, const std::string &urcHead = std::string());
+
         static constexpr std::string_view head = "+CUSD";
-        static auto isURC(const std::string &uHead) -> bool
-        {
-            return uHead.find(Cusd::head) != std::string::npos;
-        }
 
-        using Urc::Urc;
+        explicit Cusd(const std::string &urcBody, const std::string &urcHead = {});
 
+        static auto isURC(const std::string &uHead) -> bool;
```

Comment:
Sugestion:
make method  `const`
`static auto isURC(const std::string &uHead) const -> bool;`

Reply:
A `static` method can't be `const` at the same time as it isn't bound to any instance of this class.